### PR TITLE
Test:Add unit test of triggerinstance controller

### PR DESCRIPTION
**Pull request by Mcduller**
I add the normal case about creating triggerinstance and relevant resource of triggerinstance controller
Signed-off-by: Mcduller <1596582524@qq.com>

### CODE REVIEW

**Comment by charlie0129** [comment]
Actually, you don't need to close your pr if the check fails (or if there any special reason that you need to close this?)

**Comment by Mcduller** [comment]
> Actually, you don't need to close your pr if the check fails (or if there any special reason that you need to close this?)

ok,I got it ,i will reopen it

**Comment by Mcduller** [comment]
@charlie0129 hi, I had fix the check error, could you review this pr ?

File: controllers/triggerinstance/suite_test.go [hunk 1]
```diff
@@ -68,10 +73,35 @@ var _ = BeforeSuite(func() {
...
+		Config:       config.Config{},
+	}
+	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
+		Scheme:                  scheme.Scheme,
+		MetricsBindAddress:      "0",
+		LeaderElection:          false,
+		LeaderElectionNamespace: "default",
+		LeaderElectionID:        "test",
+	})
+	var done = make(chan interface{})
```
[the remark this answers is not in the document]
Comment (Mcduller) [reply controllers/triggerinstance/suite_test.go:89]: > What does this `done` do?

It's redundant, i will remove it.

**Review by charlie0129 — APPROVED** [review]
Good job!

**Pull request merged**